File: interview-prep-client/app/questionAggregate/model/QuestionItem.ts
export class QuestionItem {
    private _question: string;
    private _answer: string;

    constructor(question: string, answer: string){
        this._question = question;
        this._answer = answer;
    }

    public get Question(): string{
        return this._question;
    }

    public get Answer(): string{
        return this._answer;
    }
}

File: interview-prep-api/models/Question/QuestionItem.cs
using UserModels;
using System.ComponentModel.DataAnnotations.Schema;

namespace Question.Models;

[Table("QuestionItems")]
public class QuestionItem
{
    public long Id { get; set; }
    public string? Question { get; set; }
    public string? Answer { get; set; }

    // Foreign key for User
    public long UserId { get; set; }

    // Navigation property for User
    public UserModels.User User { get; set; }
}
